kotlintest-assertions/src/main/kotlin/io/kotlintest/dsl.kt

```diff
@@ -162,7 +162,7 @@ fun diffLargeString(expected: String, actual: String, minSizeForDiff: Int = 50):
 
   val useDiff = expected.lines().size >= minSizeForDiff
       && actual.lines().size >= minSizeForDiff &&
-      System.getProperty("kotlintest.assertions.multilinediff") != "simple"
+      System.getProperty("kotlintest.assertions.multi-line-diff") != "simple"
```

Comment:
Any reason for this to be brought together? Intentional or accident?

Reply:
If you mean why did I change it on this commit, it is because I only added it last night (so not in any release yet), and when I added the flag for _this_ commit, I thought putting in the hyphens made it look better, so I did the same change here.